tests/test_issue_maker.py

```diff
@@ -0,0 +1,174 @@
+import os
+import time
+import unittest
+from unittest.mock import patch
+from test.support import EnvironmentVarGuard
+
+from requests import Response
+
+
+class TestBase(unittest.TestCase):
+    def setUp(self):
+        self.env = EnvironmentVarGuard()
```

Comment:
Tests look cool I'm just curious about this. Will we need to maintain all env vars like this? Maybe just os.environ could be patched class-wide?

Reply:
Changed that